public/src/components/epicTests/epicTestEditor.tsx

```diff
@@ -22,35 +22,36 @@ import EditableTextField from "../helpers/editableTextField"
 import { Region } from '../../utils/models';
 import EpicTestVariantsList from './epicTestVariantsList';
 import MaxViewsEditor from './maxViewsEditor';
-import { renderVisibilityIcons, renderVisibilityHelpText, renderDeleteIcon } from './utilities';
+import { renderVisibilityIcons, renderVisibilityHelpText } from './utilities';
 import {onFieldValidationChange, ValidationStatus} from '../helpers/validation';
 import ButtonWithConfirmationPopup from '../helpers/buttonWithConfirmationPopup';
 import DeleteSweepIcon from '@material-ui/icons/DeleteSweep';
+import ArchiveIcon from '@material-ui/icons/Archive';
 
 const styles = ({ spacing, typography}: Theme) => createStyles({
   container: {
-    width: "100%",
+    width: '100%',
     borderTop: `2px solid #999999`,
     marginLeft: spacing(2),
     marginTop: spacing(6)
   },
   formControl: {
     marginTop: spacing(2),
     marginBottom: spacing(1),
-    display: "block",
+    display: 'block',
   },
   h3: {
     fontSize: typography.pxToRem(28),
     fontWeight: typography.fontWeightMedium,
-    margin: "10px 0 15px 0"
+    margin: '10px 0 15px 0'
   },
   hasChanged: {
     color: 'orange'
   },
   h4: {
     fontSize: typography.pxToRem(24),
     fontWeight: typography.fontWeightMedium,
-    margin: "20px 0 15px 0"
+    margin: '20px 0 15px 0'
```

Comment:
if you prefer being succinct, this is equivalent to
```suggestion
    margin: '20px 0 15px'
```

Reply:
👍